openmeetings-web/pom.xml

```diff
@@ -627,6 +627,14 @@
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-lang3</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.prometheus</groupId>
+			<artifactId>simpleclient_servlet</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.tomcat</groupId>
+			<artifactId>tomcat-catalina</artifactId>
```

Comment:
Will this introduce any conflicts if started with Jetty?

Reply:
Jetty seems to work fine. I did test starting it up with embedded Jetty via Maven and there wasn't any issue. 
Neither during startup nor later in the openmeetings.log